tests/Unit/Domain.LinnApps.Tests/Domain.LinnApps.Tests.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>netcoreapp2.2</TargetFramework>
     <AssemblyName>Linn.Stores.Domain.LinnApps.Tests</AssemblyName>
     <RootNamespace>Linn.Stores.Domain.LinnApps.Tests</RootNamespace>
+    <LangVersion>latest</LangVersion>
```

Comment:
Should I include this? I Had to set it to un-break some tests using named params

Reply:
no idea